scripts/prog13.rb

```diff
@@ -0,0 +1,4 @@
+puts "enter number"
+num=gets
+num=num.to_i
+puts fact=num.downto(1).inject(:*)
```

Comment:
You have used Enumerator class here but the question says you should implement using ranges. It is easier that way, you don't need to use the downto method at all, just `(1..5)` would do the work.